packages/turf-transform-rotate/test/in/polygon.geojson

```diff
@@ -0,0 +1,40 @@
+{
+  "type": "Feature",
+  "properties": {
+    "angle": 200,
+    "pivot": {
```

Comment:
Noticed you applied this change to other fixtures (looks like this was the only one) 👍